go/pkg/apis/enricher/framework/java/wildfly_detector.go

```diff
@@ -0,0 +1,33 @@
+/*******************************************************************************
+ * Copyright (c) 2021 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ * Red Hat, Inc.
+ ******************************************************************************/
+package enricher
+
+import (
+	"context"
+
+	"github.com/redhat-developer/alizer/go/pkg/apis/model"
+)
+
+type WildFlyDetector struct{}
+
+func (o WildFlyDetector) GetSupportedFrameworks() []string {
+	return []string{"WildFly"}
+}
+
+func (o WildFlyDetector) DoFrameworkDetection(language *model.Language, config string) {
+	if hasFwk, _ := hasFramework(config, "org.wildfly"); hasFwk {
```

Comment:
Sorry for the obviious question there... I am wandering how you do detect that it is WildFly? The maven plugin in use is "org.wildfly.plugins:wildfly-maven-plugin". org.wildfly seems very generic. What are you checking actually?

Reply:
I've tried to have the same logic as the rest of the detectors for `java_recognizer`. I think it is better to have a more specific `org.wildfly.plugins` as a `tag` inside [IsTagInPomXMLFile](https://github.com/thepetk/alizer/blob/ce9ab6dfd53e07177519dc63683610980c80a62e/go/pkg/utils/detector.go#L85)

Reply:
I've updated both `detectors` to check for the full `group_id` following the same logic for all the other `detectors` of the `go` implementation.